.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: whatwg/html-build
+    steps:
+    - name: Checkout whatwg/html-build
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Shellcheck
+      run: |
+        shellcheck *.sh
```

Comment:
How about just `shellcheck **/*.sh`?

Reply:
This ends up with lots of errors in `entities/build.sh`, which I guess we're leaving untouched for now.

Reply:
Ah, OK